src/databricks/sql/utils.py

```diff
@@ -478,11 +478,15 @@ def _create_arrow_array(t_col_value_wrapper, arrow_type):
 
 
 class DbSqlType(Enum):
+    """The values of this enumeration are passed as literals to be used in a CAST
+    evaluation by the thrift server.
+    """
+
     STRING = "STRING"
     DATE = "DATE"
     TIMESTAMP = "TIMESTAMP"
     FLOAT = "FLOAT"
-    DECIMAL = "DECIMAL"
+    DECIMAL = "DECIMAL(6,2)"
```

Comment:
If we keep this as just `DECIMAL` then DBR will create a decimal with zero precision after the decimal point because in Databricks SQL

```sql
SELECT CAST("1234.56" as DECIMAL)
```

returns `1235`

Reply:
As of 1565700 I've implemented dynamic casting and removed the `DECIMAL(6,2)` default